docs/src/counting.md

```diff
@@ -6,66 +6,73 @@ end
 ```
 
 # Counting
+`BioSequences` contain functionality to efficiently count biosymbols in a biosequence
```

Comment:
If this is referring to the package, i think it should use the singular verb (eg `Biosequences.jl` contains") - if it refers to multiple instances of a type, it should maybe be `BioSequence`s (the "s" outside the ticks)

Reply:
This always trips me up in English - despite being fluent, I'm not at the point where these things automatically sound right or wrong :sweat_smile: